public/_redirects

```diff
@@ -0,0 +1 @@
+/*    /index.html   200
```

Comment:
ça sert à réactualiser Netlify quand on deploy ?

Reply:
A quoi cela sert il ?

Reply:
Ca sert a ne pas avoir le 404 not found quand on actualise la page